src/main/java/com/example/demo/matcher/services/TradeService.java

```diff
@@ -0,0 +1,22 @@
+package com.example.demo.matcher.services;
+
+import com.example.demo.matcher.models.Trade;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TradeService {
```

Comment:
Is it possible to add an `@Getter` before the trades and omit the `get()` method?